### ISSUE TITLE
Failures on CFE unittest of flutter widget incremental dill when changing to 2.10

### ISSUE THREAD

**Issue by whesse** [dart-lang/sdk#42794]
There are new test failures on [Update tests that have the current version number hardcoded...Increase Dart version to 2.10](https://github.com/dart-lang/sdk/compare/f88ce7aef5f8a41fddfab8e26f8b2c6e0e950236~...24c7666def53832a31559beac130eb3336008110).

The tests
```
pkg/front_end/test/incremental_load_from_dill/flutter_widget_transform_nnbd Fail (expected Pass)
pkg/front_end/test/incremental_load_from_dill/flutter_widget_transformer_non_const Fail (expected Pass)
standalone/io/https_client_certificate_test RuntimeError (expected Pass)
standalone/io/secure_socket_renegotiate_test RuntimeError (expected Pass)
```
are failing on configurations
```
cfe-unittest-asserts-release-linux
cfe-unittest-asserts-release-mac
cfe-unittest-asserts-release-win
dartk-strong-linux-debug-x64
dartk-strong-linux-release-x64
```

This may be an existing problem in the test code, and just a mismatch in golden files.  Will take a look at it.
Most likely cause would be a 2.9 language version that needs to be changed to 2.10.

**Comment by whesse** [comment]
Yes, this code is not being compiled in null-safe mode, when it has null-safe syntax.  Something needs to be updated.

**Comment by johnniwinther** [comment]
`pkg/front_end/test/incremental_load_from_dill/flutter_widget_transform_nnbd` and
`pkg/front_end/test/incremental_load_from_dill/flutter_widget_transformer_non_const` have been fixed in https://dart-review.googlesource.com/c/sdk/+/155600
